tests/apm/mod.rs

```diff
@@ -54,7 +54,7 @@ fn read_first_non_monitor_line(file: &mut BufReader<&File>, line: &mut String) {
 
 #[test]
 fn logging() {
-    let _ = fs::remove_file("test_apm_log.txt");
+    let _ = fs::remove_file("test_log.txt");
```

Comment:
That might have been my fault -- i changed a lot of stringy fields to be more specific when I was parallelizing all of the tests.